tests/unit/libbpf_test.cpp

```diff
@@ -2899,8 +2899,10 @@ TEST_CASE("recursive_tail_call", "[libbpf]")
     }
 
     // Verify that the printk output is correct.
-    REQUIRE(output.size() == MAX_TAIL_CALL_CNT);
-    for (size_t i = 0; i < MAX_TAIL_CALL_CNT; i++) {
+    // In recurse ebpf program, the printk is added even to the caller.
```

Comment:
I can't parse "In recurse ebpf program"

Reply:
In tail_call_recursive.c, the recurse program:

SEC("xdp_prog") int recurse(struct xdp_md* ctx)

Reply:
Update comment to clarify.  E.g.,:
In the "recurse" BPF program, ...

Reply:
Fixed.